### PR TITLE
Setup Docusaurus website

### LINKED ISSUE #36: Add documentation microsite
## requirements
Add documentation microsite.
It should include both backend and frontend docs.

## specs
{optional: describe technical specs to implement this feature, if not obvious}

## misc
{optional: other useful info}

### PR DESCRIPTION
**Pull request by gabro**
Closes #36 

See preview at https://buildo.github.io/toctoc/

image

The content is a porting of our internal Paper + installation instructions.

We can tackle the guides and examples later on.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -66,4 +69,15 @@ lazy val ldap = project
...
 
-parallelExecution in Test := false
+lazy val docs = project
+  .in(file("toctoc-docs"))
+  .settings(
+    skip.in(publish) := true,
+    moduleName := "toctoc-docs",
+    mdocVariables := Map(
+      "VERSION" -> version.value,
+      "STABLE_VERSION" -> version.value.replaceFirst("\\+.*", ""),
```
Comment (bytecodeguru) [inline build.sbt]: What does this regex do?

File: website/core/Footer.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
```
Comment (bytecodeguru) [inline website/core/Footer.js]: Are we going to fulfil this license stuff?

Comment (gabro) [reply website/core/Footer.js]: Good catch! Added a license file

File: build.sbt (hunk 1 above)
Comment (gabro) [reply build.sbt]: it converts `0.2.0+4-c9659154+20190513-1834-SNAPSHOT` to `2.0.0`

**Pull request merged**